README.rst

```diff
@@ -109,6 +109,22 @@ Note that the second parameter of ``drawToFile`` can be any
 `Python file object`_, like a ``BytesIO`` buffer if you don't want the result
 to be written on disk for example.
 
+You can convert a svg file to pillow image object as following snippet:
+
+.. code:: python
+
+    from svglib.svglib import svg2rlg
```

Comment:
Nitpicking, but this order would be a more recommended one: stdlib, external, and local packages, not split with empty lines here for brevity:

```python
import io
from PIL import Image
from reportlab.graphics import renderPM
from svglib.svglib import svg2rlg
```